align/pnr/checkers.py

```diff
@@ -138,8 +145,11 @@ def f( gen, value, tag=None):
         if found:
             with pth.open( "rt") as fp:
                 d = json.load( fp)
-            # Scale to PnRDB coords (seems like 10x um, but PnRDB is 2x um, so divide by 5
-            rational_scaling( d, div=5, errors=errors)
+            # PnRDB coordinates are in units of 2nm. Scale primitives to this unit.
+            if scale_factor == 1:
+                rational_scaling( d, mul=2, errors=errors)
+            else:
```

Comment:
You have to be careful here. For this to work you need `scale_factor` to be divisible by 2, which isn't check here at least.
You should use:
```
rational_scaling( d, mul=2, div=scale_factor, errors=errors)
```
Then you don't need the `if scale_factor == 1:` statement as well.

Reply:
Thanks, I will update. There is an assertion for scale_factor on line 42: 
`assert scale_factor == 1 or scale_factor % 2 == 0, f'PDK ScaleFactor should be even.'`